components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2014,6 +2014,10 @@ private String handleOAuthAuthorizationRequest(OAuthMessage oAuthMessage)
             validateNonceParameter(params.getNonce());
         }
 
+        if (isFapiConformant(params.getClientId())) {
+            EndpointUtil.validateFAPIResponseMode(params.getResponseType(), params.getResponseMode());
```

Comment:
Also doesn't the spec say to validate having `code id_token` in https://openid.net/specs/openid-financial-api-part-2-1_0.html#authorization-server `2.1`?

Reply:
Fixed in a9450470a49c23ea24851d9fb636453829c624e9